packages/itmat-interface/test/serverTests/users.test.ts

```diff
@@ -604,7 +628,7 @@ describe('USERS API', () => {
                     query: print(EDIT_USER),
                     variables: {
                         id: 'fakeid2',
-                        password: 'admin',
+                        password: 'admin',                        
```

Comment:
There is a whole bunch of empty spaces added here

Reply:
done